src/main/java/frc/robot/end_effector/EndEffectorConstants.java

```diff
@@ -0,0 +1,19 @@
+package frc.robot.end_effector;
+
+public class EndEffectorConstants {
+    
+    public static final int CORAL_INTAKE_CAN = 0; //TODO Coral Intake CAN ID
+    public static final double CORAL_INTAKE_REDUCTION = (36/56);
```

Comment:
When you're calculating reductions, you need to be careful to not do integer division. https://stackoverflow.com/questions/37795248/integer-division-in-java